packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -169,7 +169,7 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
     default: null,
   },
   globalFolder: {
-    description: `Folder where are stored the system-wide settings`,
+    description: `The path where all system-global files are stored.`,
```

Comment:
Shouldn't the beginning remains "Folder where [...]", since that's how the other folder settings are worded?

Reply:
That makes sense